src/controller/CHIPDeviceController.h

```diff
@@ -237,6 +242,15 @@ class DLL_EXPORT DeviceController : public Messaging::ExchangeDelegate,
 
     virtual void ReleaseDevice(Device * device);
 
+    /**
+     * @brief
+     *   Register a delegate to get notified when a device's address gets updated
+     *
+     * @param[in] delegate  The delegate of type chip::Controller::DeviceAddressUpdateDelegate
```

Comment:
`@param` comment seems to just restate the code: it is obvious that 'register*' does a registration and the code shows the type.

Could we remove extra text in comments? I would drop even `@brief` lines since they seem to just take up vertical space.

Reply:
I can drop these since as you said, the comments don't provide a lot of value.